README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
Great, thanks @GordonDiggs.  That seems to have worked.  Closing PR, I'll do other cleanups I find after I get a good build going.